openeo/rest/connection.py

```diff
@@ -1355,3 +1356,37 @@ def paginate(con: Connection, url: str, params: dict = None, callback: Callable
         page += 1
         params = {}
 
+
+
+#
+# For caching connections
+#
+
+_default_url = "openeo-dev.vito.be"
```

Comment:
we can't put a VITO specific URL like this here

Reply:
Code for connection caching was no longer relevant, so I removed it.